### DISCUSSION TITLE
How do I access json?

### DISCUSSION THREAD

**Discussion by Deninc** [jxlsteam/jxls#221] [Q&A] [answered]
I'm using jx:each(items="jdbc.query()") from Postgres.

The table has a json column, what would it be converted to in Java? Is it an object, a string..?

I tried the accessing methods like foo['bar'], foo.get('bar') and they didn't work. 

Is there any workaround? Thanks in advance.

**Comment by SoltauFintel** [comment, marked as answer]
JXLS (or JEXL) has no JSON support. Here's a solution:
```
public class JsonTool {
   public String getJsonField(String json, String fieldName) {
   // implementation ...
}}
context.put("jsonTool", new JsonTool()); // add tool to JXLS context

// in Excel cell call method on that tool class instance:
${jsonTool.getJsonField(foo, 'bar')}

```
